README.md

```diff
@@ -170,6 +170,16 @@ If you wish to make larger changes we generally recommend [forking](https://help
 
 ### Build Hets using Stack
 * [Install Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade) (use the generic Linux option if you are on Ubuntu).
+* Install build- and GUI-dependencies
+  * Ubuntu:
+    ```
+    sudo apt-get install libglib2.0-dev libcairo2-dev libpango1.0-dev libgtk2.0-dev libglade2-dev libncurses-dev
+    ```
```

Comment:
I have no idea what it is used for in Hets and I don't want to decide whether or not to include this (and thus, require libncurses-dev). @tillmo, can you please give us some input on this topic?